src/main/java/org/kairosdb/core/datastore/QueryQueuingManager.java

```diff
@@ -39,6 +39,16 @@
 	public static final String QUERY_COLLISIONS_METRIC_NAME = "kairosdb.datastore.query_collisions";
 	public static final String QUERY_AVAILABLE_PERMITS = "kairosdb.datastore.available_permits";
 	public static final String QUERY_QUEUE_LENGTH = "kairosdb.datastore.query_queue_length";
+	public static final String ARTIFACT_VERSION = "kairosdb.datastore.artifact.version";
```

Comment:
I think we are repeating this variables no? They should be in one place.

Reply:
Yes. However, this is in a different class. Originally this was used directly in MetricsResouce.java to add tags for opentracing spans.

Reply:
I probably wouldn't do that now as it would change the implementation of the class requiring changes in other places too.